apps/agent/includes/diagnostics/class-size-probe.php

```
@@ -302,19 +302,41 @@ public function registerPreRecurseFilter(): void
+     * @param mixed              $directoryCache Cached dir paths: array, or
+     *                                           false on a cold transient.
      * @return false|int
      */
     public function preRecurseDirsizeFilter(
         $size,
         string $directory,
-        $exclude,
-        int $maxExecTime,
-        array $directoryCache
+        $exclude = null,
+        int $maxExecTime = 0,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$maxExecTime'. (undefined)

(UnusedFormalParameter)